pkgs/data/fonts/lucide/default.nix

```
@@ -0,0 +1,33 @@
+{
+  stdenvNoCC,
+  lib,
+  fetchurl,
+}:
+stdenvNoCC.mkDerivation rec {
+  pname = "lucide";
+  version = "0.541.0";
+
+  src = fetchurl {
+    url = "https://unpkg.com/lucide-static@${version}/font/Lucide.ttf";
+    sha256 = "sha256-H8ACnTkbR4N3yYbWuteZoqfPlLu+OGm/NEfweBad8uM=";
```

```suggestion
    hash = "sha256-H8ACnTkbR4N3yYbWuteZoqfPlLu+OGm/NEfweBad8uM=";
```
SRI hashes use the `hash` attribute.